common/smart-home-app.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Stub for App class
+ */
+/// <reference types="@google/local-home-sdk" />
+
+export const AppStub = class {
+  private version: string;
+  private deviceManager: any;
+  constructor(version: string) {
+    this.version = version;
+  }
+  public setDeviceManager(deviceManager: any) {
+    this.deviceManager = deviceManager;
+  }
+  public getDeviceManager() {
+    return this.deviceManager;
+  }
+  public listen() {
+    return Promise.resolve();
+  }
+  public onExecute() {
```

Comment:
Are these abstract methods meant to be overridden?

Reply:
These methods are just missing implementation, I wasn't intending on overriding them.  The plan is eventually hook them up to other mocked components.

Reply:
If they're not implemented yet, can they be omitted in this change?

Reply:
The call to `loadHomeApp()` depends on these, but since it doesn't really do anything yet I'll remove both for this PR